packages/__docs__/src/Header/index.tsx

```diff
@@ -69,9 +69,12 @@ class Header extends Component<HeaderProps> {
     // If we select the latest version from the dropdown,
     // then navigate to the index (instructure.design/#currentHash).
     // In every other case eg.: v6,v7 navigate to --> instructure.design/v6/#currentHash
+    const rootToAdd = window.location.origin.includes('github.io')
```

Comment:
please add and explanatory comment for this. could be a TODO with something like "remove when we permanently migrate to github.io"